packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -1322,16 +1329,12 @@ export class ComboBox extends BaseComponent<IComboBoxProps, IComboBoxState> {
    * Handles dismissing (cancelling) the menu
    */
   private _onDismiss = (): void => {
-    // reset the selected index
-    // to the last valud state
-    this._resetSelectedIndex();
-
     // close the menu and focus the input
-    this.setState({ isOpen: false });
+    this._setOpenStateAndFocusOnClose(false, false);
```

Comment:
>false, fals [](start = 38, length = 11)

Can you add inline comments here?

Reply:
Will do!

Reply:
NIT: inline comments on the static booleans being passed to setOpenStateAndFocusOnClose

---
In reply to: [206741269](https://github.com/OfficeDev/office-ui-fabric-react/pull/5663#discussion_r206741269) [](ancestors = 206741269)